### PR TITLE
Update packages and re-enable dev script

### PR DESCRIPTION
**Pull request by nt1m**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,9 +24,10 @@
   ],
   "homepage": "https://github.com/FirefoxUX/icons",
   "devDependencies": {
+    "npm-run-all": "4.1.5",
     "photon-icons": "4.0.0",
-    "serve": "6.5.7",
-    "shelljs": "0.8.2",
+    "serve": "^10.1.2",
```
Comment (bwinton) [inline package.json]: I _strongly_ prefer exact versions, so please remove the `^` here.

**Comment by nt1m** [comment]
@bwinton Fixed!

**Comment by bwinton** [comment]
(I'll merge it when the status checks finish running…)

**Comment by bwinton** [comment]
Merged in another PR.

**Pull request closed without merging**